skaffold/src/main/kotlin/com/google/container/tools/skaffold/run/SkaffoldRunConfigurations.kt

```diff
@@ -39,6 +39,9 @@ class SkaffoldSingleRunConfiguration(
     name: String
 ) : AbstractSkaffoldRunConfiguration(project, factory, name) {
 
+    /** Single run tail logs option to stream logs after deployment until stopped. */
+    var tailDeploymentLogs: Boolean = false
```

Comment:
does this mean that it defaults to unchecked? wdyt about defaulting it to tail? also maybe add a unit test to confirm the default value (whatever we decide on).

Reply:
Sure, good idea on unit test.
As for the default value, this is derived from Skaffold approach - they do not tail by default for `run`, probably makes sense for us to keep some degree of familiarity for existing Skaffold users?

Reply:
sure. sgtm. We can always change it later if it turns out to be more useful.